spec/features/request_options_spec.rb

```diff
@@ -109,7 +84,7 @@
 
   describe 'Aeon location', js: true do
     before do
-      visit '/catalog/7916044'
+      visit '/catalog/99118600973506421'
```

Comment:
Question: Why is this not 9979160443506421?

Reply:
The test -- which in a way is redundant because we are testing on top as well for Reading Room Request button -- is looking for Reading room request button which means AEON. Both ids have a Reading Room Request button and will direct the user to Aeon. The id I used was in the updated fixtures.